File: Python, SQL, HTML, CSS, Flask, Javascript/pset6/problems-2020-x-sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

for i in range(n):
    for j in range(n-(i+1)):
        print(" ", end="")

    for j in range(i+1):
        print("#", end="")

    print(" ", end="")

    for j in range(i+1):
        print("#", end="")

    print("")

File: C/pset1/problems-2020-x-mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    //Memaksa User untuk hanya memasukkan angka 1 sampai 8
    do
    {
        n = get_int("Positive Number: ");
    }
    while (n < 1 || n > 8);
    //Loop per barisan
    for (int i = 0; i < n; i++)
    {
        //Loop untuk spasi
        for (int j = 0; j < n - (i + 1); j++)
        {
            printf(" ");
        }
        //Loop untuk # kiri
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        //Membuat Gap
        printf("  ");
        //Loop untuk # kanan
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        //Melanjutkan ke baris baru
        printf("\n");
    }
}